lib/constructs/interface.js

```diff
@@ -185,7 +221,7 @@ class Interface {
         definingInterface = member[1];
         continue;
       }
-      if (forbiddenMembers.has(member.name)) {
+      if (forbiddenMembers.has(member.name) || member.name && member.name[0] === "_") {
```

Comment:
It's worth adding a check that our IDL parsing library properly strips the `_`s from source files before handing them to us. I.e., `attribute boolean _foo` should not trigger an error.